src/app/modules/time-entries/pages/time-entries.component.ts

```diff
@@ -139,15 +139,19 @@ export class TimeEntriesComponent implements OnInit, OnDestroy {
     this.store.pipe(select(getTimeEntriesDataSource)).subscribe(ds => {
       const dataToUse = ds.data.find(item => item.project_id === event.projectId);
       if (dataToUse && this.isNewEntry()) {
-        const startDate = new Date(new Date().setHours(0, 0, 0, 0));
+        const defaultSeconds = 0;
+        const currentDate = new Date();
+        currentDate.setSeconds(defaultSeconds);
+        currentDate.setMilliseconds(defaultSeconds);
+        const emptyDate = new Date(new Date().setHours(0, 0, 0, 0));
```

Comment:
GIven that currentDate is the same that emptyDate, we can remove emptyDate and use currentDate for end_date too.